IFC4x3/Properties/s/SignalPostTelephoneType_0TSpUzg5H3AAZB2WOvWebT/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SignalPostTelephoneType_0TSpUzg5H3AAZB2WOvWebT" Name="SignalPostTelephoneType" UniqueId="1d7337bd-a854-4328-a8cb-0a063982895d" PrimaryDataType="IfcLabel" />
```

Comment:
Typing?

Reply:
This is type of signal post telephone for the signal.